beacon-chain/flags/base.go

```diff
@@ -11,7 +11,7 @@ var (
 	// HTTPWeb3ProviderFlag provides an HTTP access endpoint to an ETH 1.0 RPC.
 	HTTPWeb3ProviderFlag = &cli.StringFlag{
 		Name:  "http-web3provider",
-		Usage: "A mainchain web3 provider string http endpoint",
+		Usage: "A mainchain web3 provider string http endpoint. Can contain auth header as well in the format --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Basic xxx\" for project secret(base64 encoded) and --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Bearer xxx\" for jwt use",
```

Comment:
```suggestion
		Usage: "A mainchain web3 provider string http endpoint. Can contain auth header as well in the format --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Basic xxx\" for project secret (base64 encoded) and --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Bearer xxx\" for jwt use",
```

Reply:
I added a space before `(base64 encoded)`. It's hard to see :-)

Reply:
From this comment I would understand that we support only `Basic` and `Bearer`, but that's not the case - anything after the comma will get appended to the `Authorization` header. Can you maybe add a "for example" somewhere? 

Maybe something like this?
> "A mainchain web3 provider string http endpoint. Can contain auth header as well in the format --http-web3provider=\"[endpoint],[authorization header value]\", for example --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Basic xxx\" for project secret (base64 encoded) and --http-web3provider=\"https://goerli.infura.io/v3/xxxx,Bearer xxx\" for jwt use"